src/windows/CameraProxy.js

```diff
@@ -710,7 +710,10 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
     var UIMaxRes = WMCapture.CameraCaptureUIMaxPhotoResolution;
     var totalPixels = targetWidth * targetHeight;
 
-    if (totalPixels <= 320 * 240) {
+    if (targetWidth == -1 && targetHeight == -1) {
```

Comment:
Should we just initialize `maxRes` to higestAvailable on declaration instead of putting another if?